### ISSUE TITLE
Danger should be able to perform actions on Github

### ISSUE THREAD

**Issue by deecewan** [danger/danger-js#477]
Danger's purpose is 'stop saying you forgot to...'.
In general, this is by automating the 'you forgot to...' message.

But if the item in question is resolvable programmatically, I think Danger should have the capability to fix this.

For instance, if under certain conditions in a pull request, a certain label needs to apply, then Danger should be able to add the label. If the body of a pull request is missing something and it can be determined programatically, then Danger should be able to add it.

tl;dr Danger should be able to 'stop saying you forgot to...' by just fixing the issue, instead of making a comment saying 'you forgot to...'

I propose making the danger Github API have some write functionality (adding labels, modifying the PR body, assigning people, etc).

Edit: this is all already possible using the `danger.github.api` item. My bad 👎

**Comment by orta** [comment]
All of that is available via `danger.github.api` today

**Comment by orta** [comment]
Perhaps the reference docs for it could be improved with a link to the node docs? http://danger.systems/js/reference.html#GitHub

**Comment by deecewan** [comment]
Yeah, I realised this as I was writing my Dangerfile after having just added the `github` module to my project for this purpose.

Yeah, the GitHub reference doesn't actually point anywhere for me. I know now that it's an instance of the `github` npm package.
